src/inspector_agent.cc

```diff
@@ -242,6 +248,23 @@ class ChannelImpl final : public v8_inspector::V8Inspector::Channel,
     }
     runtime_agent_->disable();
     runtime_agent_.reset();  // Dispose before the dispatchers
+    network_agent_.reset();  // Dispose before the dispatchers
+    node_network_agent_.reset();  // Dispose before the dispatchers
+  }
+
+  void emitNotificationFromBackend(const StringView& event,
+                                   const StringView& params) {
+    std::unique_ptr<protocol::DictionaryValue> value =
+        protocol::DictionaryValue::cast(
+            protocol::StringUtil::parseJSON(params));
+    std::string raw_event = protocol::StringUtil::StringViewToUtf8(event);
+    std::string domain_name = raw_event.substr(0, raw_event.find('.'));
+    std::string event_name = raw_event.substr(raw_event.find('.') + 1);
+    if (domain_name == "Network") {
```

Comment:
Should `Network` domain be exposed to `inspector.emitProtocolEvent`?

Reply:
The `Network` domain is internally required by the `NodeNetwork` domain but is not directly available from the inspector's API.